sandbox-open-imis/charts/backend/templates/deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.deployment.name }}
+  namespace: {{ .Values.deployment.namespace }}
+spec:
+  selector:
+    matchLabels:
+      run: {{ .Values.deployment.spec.name }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: {{ .Values.deployment.spec.name }}
+    spec:
+      containers:
+        - name: {{ .Values.deployment.containers.name }}
+          image: {{ .Values.deployment.containers.image }}
```

Comment:
I think convention is to use {{ .Values.image }} directly unless you have multiple images in same deployment

Reply:
Good point too. 
But before applying it need to have team opinion regards other similar cases.
https://govstack-global.atlassian.net/browse/SND-205
Thank you for a feedback.